ly/musicxml/ly2xml_mediator.py

```diff
@@ -353,6 +354,17 @@ def new_key(self, key_name, mode):
         else:
             self.current_attr.set_key(get_fifths(key_name, mode), mode)
 
+    def new_mark(self):
+        if self.bar is None:
+            self.new_bar()
+        if self.bar.has_music():
+            new_bar_attr = xml_objs.BarAttr()
+            new_bar_attr.set_mark(self.current_mark)
+            self.add_to_bar(new_bar_attr)
+        else:
+            self.current_attr.set_mark(self.current_mark)
+        self.current_mark = chr(ord(self.current_mark) + 1)
```

Comment:
Currently this solution just adds marks correctly from `A` to `Z` (I wasn't aware of the `I` skipping). I'm working to make it able to add `AA` after `Z` and so on.